diff -ur SGDK-orig/makefile.gen SGDK/makefile.gen
--- SGDK-orig/makefile.gen	2021-03-12 23:26:53.000000000 +0100
+++ SGDK/makefile.gen	2021-03-12 23:26:53.000000000 +0100
@@ -16,21 +16,20 @@
 RES := res
 INCLUDE := inc
 
-SHELL := $(BIN)/sh
-RM := $(BIN)/rm
-CP := $(BIN)/cp
-CC := $(BIN)/gcc
-LD := $(BIN)/ld
-NM := $(BIN)/nm
+RM := rm
+CP := cp
+CC := m68k-elf-gcc
+LD := m68k-elf-ld
+NM := m68k-elf-nm
 JAVA := java
 ECHO := echo
-OBJCPY := $(BIN)/objcopy
-ASMZ80 := $(BIN)/sjasm
-MACCER := $(BIN)/mac68k
-SIZEBND := $(BIN)/sizebnd
-BINTOS := $(BIN)/bintos
-RESCOMP := $(JAVA) -jar $(BIN)/rescomp.jar
-MKDIR := $(BIN)/mkdir
+OBJCPY := m68k-elf-objcopy
+ASMZ80 := sjasm
+MACCER := mac68k
+SIZEBND := sizebnd
+BINTOS := bintos
+RESCOMP := rescomp
+MKDIR := mkdir
 
 SRC_C = $(wildcard *.c)
 SRC_C += $(wildcard $(SRC)/*.c)
diff -ur SGDK-orig/makelib.gen SGDK/makelib.gen
--- SGDK-orig/makelib.gen	2021-03-12 23:26:53.000000000 +0100
+++ SGDK/makelib.gen	2021-03-12 23:26:53.000000000 +0100
@@ -12,19 +12,18 @@
 RES_LIB := $(GDK)/res
 INCLUDE_LIB := $(GDK)/inc
 
-SHELL := $(BIN)/sh
-RM := $(BIN)/rm
-CP := $(BIN)/cp
-AR := $(BIN)/ar
-CC := $(BIN)/gcc
+RM := rm
+CP := cp
+AR := m68k-elf-ar
+CC := m68k-elf-gcc
 JAVA := java
 ECHO := echo
-OBJCPY := $(BIN)/objcopy
-ASMZ80 := $(BIN)/sjasm
-SIZEBND := $(BIN)/sizebnd
-BINTOS := $(BIN)/bintos
-RESCOMP := $(JAVA) -jar $(BIN)/rescomp.jar
-MKDIR := $(BIN)/mkdir
+OBJCPY := m68k-elf-objcopy
+ASMZ80 := sjasm
+SIZEBND := sizebnd
+BINTOS := bintos
+RESCOMP := rescomp
+MKDIR := mkdir
 
 SRC_LIB_C := $(wildcard $(SRC_LIB)/*.c)
 SRC_LIB_S := $(wildcard $(SRC_LIB)/*.s)
@@ -101,11 +100,11 @@
 
 
 $(LIB)/libmd.a: cmd_
-	$(AR) rs $(LIB)/libmd.a --plugin=liblto_plugin-0.dll @cmd_
+	$(AR) rs $(LIB)/libmd.a --plugin=/opt/sgdk/libexec/gcc/m68k-elf/9.3.0/liblto_plugin.so @cmd_
 	$(RM) cmd_
 
 $(LIB)/libmd_debug.a: cmd_
-	$(AR) rs $(LIB)/libmd_debug.a --plugin=liblto_plugin-0.dll @cmd_
+	$(AR) rs $(LIB)/libmd_debug.a --plugin=/opt/sgdk/libexec/gcc/m68k-elf/9.3.0/liblto_plugin.so @cmd_
 	$(RM) cmd_
 
 cmd_ : $(OBJ_LIB)
